pkg/build/builder/docker.go

```diff
@@ -416,6 +426,49 @@ func traverseAST(cmd string, node *parser.Node) int {
 	return index
 }
 
+// insertEnvAfterFrom inserts an ENV instruction with the environment variables
+// from env after every FROM instruction in the given Dockerfile instructions
+// and returns a new Dockerfile.
+func insertEnvAfterFrom(instructions []byte, env []kapi.EnvVar) ([]byte, error) {
+	if len(env) == 0 {
+		return instructions, nil
+	}
+	node, err := parser.Parse(bytes.NewReader(instructions))
+	if err != nil {
+		return nil, err
+	}
+
+	// buildEnv is an ENV Dockerfile instruction to define all environment
+	// variables from env.
+	buildEnv := []byte("ENV")
```

Comment:
you don't have to uppercase, just use the constant. We are not going to publish this Dockerfile as it is just an internal object we build the image from. But i'm fine either by what you have or if you already imported the package with constants, use the constants ;-)